Userland/Libraries/LibWeb/Painting/GradientPainting.cpp

```diff
@@ -66,7 +66,7 @@ static ColorStopData resolve_color_stop_positions(auto const& color_stop_list, a
     //    or transition hint before it.
     auto max_previous_color_stop_or_hint = resolved_color_stops[0].position;
     auto resolve_stop_position = [&](auto& position) {
-        float value = resolve_position_to_float(position);
+        float value = static_cast<float>(resolve_position_to_float(position));
```

Comment:
As a purely semantics thing I think the callbacks in the `resolve_X_gradient_data()` functions should be adjusted to return floats again (not CSSPixels, which I think is why this cast was added), since value returned really is not in any type of pixels.

Reply:
Oops, my bad. My continued lack of understanding of gradient code keeps showing. :sweat_smile: